### ISSUE TITLE
productVariantValue "no Gala-Show-Ticket"

### ISSUE THREAD

**Issue by smadamit** [inbaz/ers#216]
If we have the package week ticket including gala show we need the productVariantValue "no Gala-Show-Ticket". With this option the user can order a week ticket without a gala show (maybe he leaves the EJC before Friday). But now there is this option also in the product Gala-Show-Ticket. Users can put in Gala-Show-Ticket with option "no Gala-Show-Ticket". Not a big deal, but we have to keep this issue in mind.

Works for you?

**Comment by inbaz** [comment]
I thin if I'm implementing the subproduct functionality I will add an option to the item which already says no "sub product name". Maybe this can be made configurable through the admin panel if the no "..." option will be there. That would be the easiest and clearest way. I don't like that when you can buy a normal gala show ticket and select no gala show.

**Comment by smadamit** [comment]
If you implement it, very nice, than I discard the option "no Gala-Show-Ticket".

**Comment by philip-n** [comment]
Do we really need the non-gala-show-ticket option? I'd argue that everybody who pays the full week price is entitled to a gala show ticket. If they don't want it, then they can always give it to someone else (or just don't take it at check-in).

From the "Can it cause a problem"-point of view:
* If people select the no-ticket-option and then decide different (or select it by accident), it can be problematic
* If people select a gala show and don't want to have the ticket there's no problem (as I guess that the number of people affected is small enough to not matter for EJC organizers in terms of show capacity)

So I vote for just removing the option. Otherwise, I'd put the option as the last one to prevent selection by accident.

**Comment by smadamit** [comment]
@philip-n at the moment the "no gala show" option is a workaround and a config from the admin panel. Andi is planning to implement it:
- product: galashow ticket -> you can select the date: Fr, Sa 18:00 or Sa 21:00 -> there is no option "no gala"
- product package week ticket including gala show: you can select Fr, Sa 18:00 or Sa 21:00 and on top you have the choice to put no gala ticket into the card

I think the real problem from EJC-orga side: What is if the amount of gala tickets is running out? And than you have maybe several tickets which will not be used? -> e.g. very young kids/babys...

**Comment by philip-n** [comment]
@smadamit : Full solution sounds nice. My idea was rather meant for avoiding work now as I don't see this issue as a too big one.

Regarding show capacity: If seats are limited and running out, I think that people who took a show ticket which they don't need will give it to someone else during their stay at EJC. And I guess that we are rather talking about 50 than 500 jugglers affected.

Talking about small children, we usually have the rule that they can enter the show for free, without a ticket, if they don't need a seat of their own. If they do need a seat (and fall into the "free entry" age group), they have to buy a show ticket.
--> Don't issue show tickets to age group 0-5 years.

In summary, I still think that neglecting the "no ticket" option wouldn't hurt :-)

**Comment by smadamit** [comment]
Don't issue show tickets to age group 0-5 years. -> that would produce work at another point. See also my email.

**Comment by philip-n** [comment]
Thanks for the information (here and by mail)!

**Comment by inbaz** [comment]
So, please test in 0.2.65

**Comment by smadamit** [comment]
0.2.65 -> OK
